### PR TITLE
fix(combobox): sync styles with Select styles

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Set input right padding to `space100`
- [x] Use `SelectIconWrapper` for chevron

### CODE REVIEW

File: packages/paste-core/components/combobox/src/Combobox.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const ComboboxIconWrapper: React.FC = ({children}) => (
     display="inline-flex"
     position="absolute"
     pointerEvents="none"
-    right="space40"
+    right="space30"
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-core/components/combobox/src/Combobox.tsx:20]: Yeah I think that makes sense. We have a ticket in the backlog to split the form package into individual components with a potential utils package for label. Should we do that as part of that ticket or go ahead and export just this component from forms now?

[the remark this answers is not in the document]
Comment (SiTaggart) [reply packages/paste-core/components/combobox/src/Combobox.tsx:20]: For now, this small duplication is fine. Lets get shared stuff out in the other ticket

[the remark this answers is not in the document]
Comment (TheSisb) [reply packages/paste-core/components/combobox/src/Combobox.tsx:20]: > We have a ticket in the backlog to split the form package into individual components with a potential utils package for label. Should we do that as part of that ticket or go ahead and export just this component from forms now?

I think that's a bigger ask that should be scoped separately actually. What I meant here was simply exporting the props from the select file's `SelectIconWrapper`

[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-core/components/combobox/src/Combobox.tsx]: I've gone ahead and exported `SelectIconWrapper` in the form package and used that for Comboxbox.

**Pull request merged**